src/index.html

```diff
@@ -1,12 +1,377 @@
 <!doctype html>
 <html lang="en">
+  <!-- #region head -->
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="is-active nav__link">Apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+          <li class="nav__item" data-qa="nav-hover"><a href="#laptop&computers" class="nav__link">Laptops & computers</a></li>
+          <li class="nav__item"><a href="#gadgets" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#tablets" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#photo" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#video" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+
+    <!-- #region main -->
+    <main class="main">
+      <div class="container">
+        <div data-qa="card" class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="card__image"
```

Comment:
Bad naming for alt attribute
https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#seo